libagbsyscall/Makefile

```diff
@@ -58,12 +58,21 @@ ifneq ($(PLATFORM),gba)
     CPP       := $(PREFIX)cpp
     CPPFLAGS  := -I ../include -D PORTABLE=1
     CC1       := $(PREFIX)gcc
-    CC1FLAGS  := -Wall -O2 -x c -S
+    CC1FLAGS  := -Wall -x c -S
     LD        := $(PREFIX)ld
 
+    ifeq ($(DEBUG),1)
+        CC1FLAGS += -g -O0
```

Comment:
How well does the game run without optimisation lol

Reply:
Not too bad actually! The stuff that runs does so without slowdown.
Though the emulator shows a much higher CPU usage. :D